source/Cosmos.IL2CPU/AppAssembler.cs

```diff
@@ -63,12 +63,8 @@ public AppAssembler(CosmosAssembler aAssembler, TextWriter aLog, string aLogDir)
 
         public void Dispose()
         {
-            if (mLog != null)
-            {
-                mLog.Dispose();
-                mLog = null;
-            }
-            DebugInfo.Dispose();
+            mLog?.Dispose();
```

Comment:
Why is mLog no longer set to null?

Reply:
Why would it need to be? There's no point in putting it to null, it doesn't free any extra memory